rawfile/rawfile_servicer.py

```diff
@@ -208,14 +223,19 @@ def CreateVolume(self, request, context):
                 grpc.StatusCode.INVALID_ARGUMENT, "Topology key not found... why?"
             )
 
-        params = normalize_parameters(request.parameters)
-        thin_provision = params.get("thinprovision", "no")
         format_options = params.get("formatoptions", "").strip()
+        snap_inuse = str_to_bool(params.get("snapinuse", "no"))
```

Comment:
what does this mean?

Reply:
This option controls if we are able to take snapshot from an in use volume or not

Reply:
We can also set this automatically based on COW and fsfreeze

Reply:
done.